### ISSUE TITLE
Drag 'n Drop Sort

### ISSUE THREAD

**Issue by Spooonky** [rex706/SAM#101]
Hey,
It would be very helpful if you could insert a sort using drag and drop. In this way, relevant accounts could be pushed up.

**Comment by rex706** [comment]
Yeah this is basically #23. I've put this on the back burner for a while but I will think about revisiting it.
